### DISCUSSION TITLE
Autogenerate agents and tasks

### DISCUSSION THREAD

**Discussion by yanniedog** [crewAIInc/crewAI#1108] [Feature Requests]
Would be great to leverage an LLM to populate the parameters for all the agents and tasks.

The basic idea is here, but it needs polishing:
https://github.com/yanniedog/crewai-autocrew

**Comment by PiotrEsse** [comment]
Thats ......Awesome.....
One thing thou - issues. It seeme your code generate too much doublequotes. :
<img width="614" alt="image" src="https://github.com/joaomdmoura/crewAI/assets/42764623/b247c6be-2f5d-43ee-8f28-6c920e6a9eb3">


Original crew definition looks different:
<img width="553" alt="image" src="https://github.com/joaomdmoura/crewAI/assets/42764623/25c77459-1986-440b-8bfa-b445a8254d05">

**Comment by yanniedog** [comment]
A new version is available (v1.1)

https://github.com/yanniedog/crewai-autocrew

**Comment by yanniedog** [comment]
It would be great to properly integrate this within the scope of the CrewAI project

**Comment by joaomdmoura** [comment]
Marking this as features accepted, I think I can make this an `Autonomous` process
